src/components/FeaturedProjects.js

```diff
@@ -0,0 +1,47 @@
+import React, { useEffect, useState } from 'react';
+import { Title, MainText, Section } from '../styles/Main'
+import { WrapProject, Wrapper, ProjectImage, ProjectTitle, Programs, ProgramsWrapper, Overlay } from '../styles/Projects'
+
+const FeaturedProjects = () => {
+  const selectedprojects = ['project-happy-thoughts', 'project-survey', 'project-guess-who', 'project-movies']
+
+  const [repos, setRepos] = useState([])
+
+  useEffect(() => {
+    fetch('https://api.github.com/users/Kristin-Larsson/repos')
+      .then((res) => res.json())
+      .then((data) => {
+        setRepos(data.filter(({ fork, name }) => fork && selectedprojects.includes(name)))
+      })
+  }, [])
+
+  return (
+    <Section>
+      <Title>Featured Projects</Title>
+      <Wrapper>
+        {repos.map((repo, index) => {
+          console.log(index)
+          return (
+            <div key={repo.id}>
+              <WrapProject>
+                <Overlay><p> {repo.name.replace(/-/g, ' ')} </p></Overlay>
+                <a href={`https://github.com/Kristin-Larsson/${repo.name}`}>
```

Comment:
I would try to make the whole project card clickable with some type of hover effect (Like increasing in scale). Right now it's not apparent that the title text is a link